### PR TITLE
Update to Cadence v0.24.0-patch.1 and Go SDK v0.26.0

### PR DESCRIPTION
**Pull request by turbolent**
Update to
- Cadence v0.24.0-patch.1, which is [v0.24.0](https://github.com/onflow/cadence/releases/tag/v0.24.0) + security fixes.
  - https://github.com/dapperlabs/cadence-internal/pull/85
  - https://github.com/dapperlabs/cadence-internal/pull/89
  - https://github.com/dapperlabs/cadence-internal/pull/90
  - Diff: https://github.com/onflow/cadence/compare/v0.24.0...v0.24.0-patch.1
- [Go SDK v0.26.0](https://github.com/onflow/flow-go-sdk/releases/tag/v0.26.0)

Other changes:
- Remove the storage map migration, it was part of the previous spork
- It seems like an integration test relies on functionality removed from the SDK: https://github.com/onflow/flow-go-sdk/blob/master/docs/migration-v0.25.0.md#migration-guide-v0250. I removed the function that depends on it and the related test

### CODE REVIEW

**Review by j1010001 — APPROVED** [review]

**Review by Kay-Zee — APPROVED** [review]

**Review by SupunS — APPROVED** [review]
LGTM!

**Review by sideninja — APPROVED** [review]

**Comment by sideninja** [comment]
> This does not build yet because of some SDK changes it seems:
> 
> ```
> cmd/collection/main.go:548:14: cannot initialize 1 variables with 2 values (typecheck)
> 	txSigner := sdkcrypto.NewInMemorySigner(sk, machineAccountInfo.HashAlgorithm)
> 	            ^
> cmd/consensus/main.go:821:14: cannot initialize 1 variables with 2 values (typecheck)
> 	txSigner := crypto.NewInMemorySigner(sk, machineAccountInfo.HashAlgorithm)
> 	            ^
> gomock_reflect_656513307/prog.go:26:37: Network not declared by package module (typecheck)
> 		{"Network", reflect.TypeOf((*pkg_.Network)(nil)).Elem()},
> ```

It now returns an error as well.

**Review by jordanschalm — APPROVED** [review]

**Comment by turbolent** [comment]
Does anyone know why the integration tests fail?

```
docker build -f cmd/Dockerfile --ssh default --build-arg TARGET=./cmd/collection --build-arg COMMIT=40cb99781f84a75bb258f2860203009c2c8e25de  --build-arg VERSION=40cb997 --build-arg GOARCH=amd64 --target production \
	--label "git_commit=40cb99781f84a75bb258f2860203009c2c8e25de" --label "git_tag=40cb997" \
	-t "gcr.io/flow-container-registry/collection:latest" -t "gcr.io/flow-container-registry/collection:40cb997" -t "gcr.io/flow-container-registry/collection:40cb997" .
could not parse ssh: [default]: invalid empty ssh agent socket, make sure SSH_AUTH_SOCK is set
```

**Review by AlexHentschel — APPROVED** [review]

File: cmd/util/cmd/execution-state-extract/execution_state_extract.go [hunk 1]
```diff
@@ -68,13 +68,7 @@ func extractExecutionState(
...
 
-		orderedMapMigration := mgr.OrderedMapMigration{
-			Log:       log,
-			OutputDir: dir,
-		}
-
 		migrations = []ledger.Migration{
-			orderedMapMigration.Migrate,
 			storageUsedUpdateMigration.Migrate,
 			mgr.PruneMigration,
```
Comment (ramtinms) [inline cmd/util/cmd/execution-state-extract/execution_state_extract.go:73]: If we don't have any other migrations, we could actually skip these two as well, should not have an impact but might make spork faster.

**Review by ramtinms — COMMENTED** [review]
looks good to me

**Pull request merged**